app/test_main.py

```diff
@@ -1,3 +1,88 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
+    @pytest.mark.parametrize(
+        "initial_cat_age,"
+        "initial_dog_age,"
+        "expected_human_age_cat,"
+        "expected_human_age_dog",
+        [
+            pytest.param(
+                0,
+                0,
+                0,
+                0,
+                id="test should return zero when age is zero"
```

Comment:
Rewrite input values as follows:
```suggestion
                0,
                0,
                [0, 0],
                id="test should return zero when age is zero"
```